ext/opentelemetry-ext-otcollector/tests/test_otcollector_metrics_exporter.py

```diff
@@ -34,25 +34,11 @@ class TestCollectorMetricsExporter(unittest.TestCase):
     @classmethod
     def setUpClass(cls):
         # pylint: disable=protected-access
-        cls._meter_defaults = (
-            metrics._METER_PROVIDER,
-            metrics._METER_PROVIDER_FACTORY,
-        )
-        metrics.set_preferred_meter_provider_implementation(
-            lambda _: MeterProvider()
-        )
+        metrics.set_meter_provider(MeterProvider())
         cls._meter = metrics.get_meter(__name__)
         kvp = {"environment": "staging"}
         cls._test_label_set = cls._meter.get_label_set(kvp)
 
-    @classmethod
-    def tearDownClass(cls):
-        # pylint: disable=protected-access
-        (
-            metrics._METER_PROVIDER,
-            metrics._METER_PROVIDER_FACTORY,
-        ) = cls._meter_defaults
```

Comment:
It seems like tests that set the meter/tracer provider on setup should still unset it on teardown. Why remove this instead of update it?